### PR TITLE
Refactor order and pattern tests approach 

### PR DESCRIPTION
**Pull request by vasylshpak**
- Refactor order tests and `pattern.query` test.
- Tests start with one command

### CODE REVIEW

File: __tests__/order/order.query.test.js [hunk 1]
```diff
@@ -137,15 +143,13 @@ describe('Order queries', () => {
         firstName: 'Test',
         phoneNumber: '380953544271',
         patronymicName: 'Test',
```
Comment (TorskyM) [inline __tests__/order/order.query.test.js:145]: I think it is better to move `Pattern` data to a variable file

File: __tests__/pattern/pattern.query.test.js [hunk 2]
```diff
@@ -138,12 +139,11 @@ describe('pattern query tests', () => {
           ...item,
         }))
       );
-      expect(res.data.getPatternById.images).toBeInstanceOf(Object);
+      //expect(res.data.getPatternById.images).toBeInstanceOf(Object);
```
Comment (TorskyM) [inline __tests__/pattern/pattern.query.test.js:142]: Remove useless comment

File: __tests__/pattern/pattern.query.test.js [hunk 3]
```diff
@@ -244,10 +244,10 @@ describe('pattern query tests', () => {
 
       expect(res.data.getAllPatterns).toMatchSnapshot();
       expect(res.data.getAllPatterns.items).toHaveLength(5);
-      expect(res.data.getAllPatterns.count).toEqual(16);
+      expect(res.data.getAllPatterns.count).toEqual(18);
     });
     test('pattern pagination test with wrong arguments', async () => {
```
Comment (TorskyM) [inline __tests__/pattern/pattern.query.test.js:249]: Fix test label it should describe test expectations

File: .env.test [hunk 4]
```diff
@@ -1,4 +1,4 @@
-MONGO_URL=mongodb://root:root@localhost:27017/horondi?authSource=admin
+MONGO_URL=mongodb://localhost:27017/?readPreference=primary&appname=MongoDB%20Compass&ssl=false
```
Comment (bandvov) [inline .env.test:1]: change this line to MONGO_URL=mongodb://root:root@localhost:27017/horondi?authSource=admin

File: __tests__/order/order.query.test.js [hunk 5]
```diff
@@ -1,13 +1,19 @@
...
 const { newOrder } = require('./order.variables');
+const { setupApp } = require('../helper-functions');
+jest.mock('../../modules/upload/upload.service');
 
 let orderId;
+let operations;
 
-describe('Order queries', () => {
+describe('Pattern queries', () => {
   beforeAll(async () => {
```
Comment (bandvov) [inline __tests__/order/order.query.test.js:11]: why did you write `pattern` instead of `order`

File: __tests__/order/order.query.test.js [hunk 6]
```diff
@@ -1,13 +1,19 @@
...
-const client = require('../../utils/apollo-test-client');
 const { newOrder } = require('./order.variables');
+const { setupApp } = require('../helper-functions');
+jest.mock('../../modules/upload/upload.service');
 
 let orderId;
+let operations;
 
-describe('Order queries', () => {
+describe('Pattern queries', () => {
```
Comment (bandvov) [inline __tests__/order/order.query.test.js:10]: why did you change `order` to `pattern`?

File: __tests__/pattern/pattern.query.test.js (hunk 2 above)
Comment (bandvov) [reply __tests__/pattern/pattern.query.test.js:142]: uncomment this line

**Review by bandvov — CHANGES_REQUESTED** [review]

**Comment by vasylshpak** [comment]
- move data to variable file
- delete unnecessary comments
- change .env.test database url
- fix test label
- delete syntax mistakes

**Review by TorskyM — APPROVED** [review]

File: __tests__/order/order.query.test.js (hunk 1 above)
Comment (vasylshpak) [reply __tests__/order/order.query.test.js]: Fixed

File: __tests__/pattern/pattern.query.test.js (hunk 2 above)
Comment (vasylshpak) [reply __tests__/pattern/pattern.query.test.js]: Done

File: .env.test (hunk 4 above)
Comment (vasylshpak) [reply .env.test:1]: Done

File: __tests__/order/order.query.test.js (hunk 6 above)
Comment (vasylshpak) [reply __tests__/order/order.query.test.js]: Done

File: .env.test [hunk 7]
```diff
@@ -1,4 +1,4 @@
-MONGO_URL=mongodb://root:root@localhost:27017/horondi?authSource=admin
```
Comment (bandvov) [inline .env.test:1]: you should return removed line

**Review by bandvov — APPROVED** [review]

File: .env.test (hunk 7 above)
Comment (vasylshpak) [reply .env.test]: done
